src/resources/locale-data-polyfill.ts

```diff
@@ -1,70 +1,36 @@
 // Loads the static locale data for a given language from FormatJS
 // Parents need to load polyfills first; they are not imported here to avoid a circular reference
 
-const loadedPolyfillLocale = new Set();
+const INTL_POLYFILLS = [
+  "DateTimeFormat",
+  "DisplayNames",
+  "ListFormat",
+  "NumberFormat",
+  "RelativeTimeFormat",
+] as const satisfies readonly (keyof typeof Intl)[];
+
+const loadedLocales: Set<string> = new Set();
+
+const addData = async (
+  obj: (typeof INTL_POLYFILLS)[number],
+  language: string
+) => {
+  // Add function will only exist if constructor is polyfilled
+  if (typeof (Intl[obj] as any)?.__addLocaleData === "function") {
+    const result = await fetch(
+      `${__STATIC_PATH__}locale-data/intl-${obj.toLowerCase()}/${language}.json`
+    );
+    // Ignore if polyfill data does not exist for language
+    if (result.status !== 404) {
```

Comment:
I presume that the sweeping try/catch was just supposed to ignore errors when the data doesn't exist, so I replaced it with just a 404 check.  Let me know if I'm missing something.

Reply:
just check `result.ok`?

Reply:
Had that originally... after some sleep I'm not even sure why I changed it 🤔